bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ExperienceFragmentImpl.java

```diff
@@ -0,0 +1,183 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2019 Adobe Systems Incorporated
+ ~
```

Comment:
Adobe changed name to Adobe.
Along there is a new License header please replace

Reply:
Addressed in commit https://github.com/adobe/aem-core-wcm-components/commit/7bce91c9a620b6d169695210a9d0e07347369a2e